### PR TITLE
Segfault Handler

### PR DESCRIPTION
**Pull request by chrisdopuch**
Adds segfault handling and stack trace printing. This will be really useful for debugging the segfault problems we have been getting. 

I also updated the shrinkwrap.json

### CODE REVIEW

**Comment by chrisdopuch** [comment]
@joelgriffith @jhsu looking for input

Joe was thinking about getting rid of the shrinkwrap. See conversation on #pdm

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,62 +1,62 @@
 {
```
Comment (joelgriffith) [inline npm-shrinkwrap.json]: Alright, I can see ditching shrinkwrap.json now...

**Comment by joelgriffith** [comment]
Let's ditch `npm-shrinkwrap.json`. Other than that :+1:

File: npm-shrinkwrap.json (hunk 1 above)
Comment (chrisdopuch) [inline npm-shrinkwrap.json]: Yeah it's pretty funky how npm on different machines can output different things to the shrinkwrap file.

**Comment by joelgriffith** [comment]
@chrisdopuch if you can remove npm-shrinkwrap and then push up your branch (it should update this PR), then we can push the greenie button.

**Pull request merged**